staticaddr/withdraw/manager.go

```diff
@@ -1026,61 +1037,6 @@ func toPrevoutInfo(outpoints []wire.OutPoint) []*staticaddressrpc.PrevoutInfo {
 	return result
 }
 
-// createMusig2Sessions creates a musig2 session for a number of deposits.
-func (m *Manager) createMusig2Sessions(ctx context.Context,
-	deposits []*deposit.Deposit) ([]*input.MuSig2SessionInfo, [][]byte,
-	error) {
-
-	musig2Sessions := make([]*input.MuSig2SessionInfo, len(deposits))
-	clientNonces := make([][]byte, len(deposits))
-
-	// Create the sessions and nonces from the deposits.
-	for i := 0; i < len(deposits); i++ {
-		session, err := m.createMusig2Session(ctx)
-		if err != nil {
-			return nil, nil, err
-		}
-
-		musig2Sessions[i] = session
-		clientNonces[i] = session.PublicNonce[:]
-	}
-
-	return musig2Sessions, clientNonces, nil
-}
-
-// Musig2CreateSession creates a musig2 session for the deposit.
-func (m *Manager) createMusig2Session(ctx context.Context) (
-	*input.MuSig2SessionInfo, error) {
-
-	addressParams, err := m.cfg.AddressManager.GetStaticAddressParameters(
-		ctx,
-	)
-	if err != nil {
-		return nil, fmt.Errorf("couldn't get confirmation height for "+
-			"deposit, %w", err)
-	}
-
-	signers := [][]byte{
-		addressParams.ClientPubkey.SerializeCompressed(),
-		addressParams.ServerPubkey.SerializeCompressed(),
-	}
-
-	address, err := m.cfg.AddressManager.GetStaticAddress(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("couldn't get confirmation height for "+
-			"deposit, %w", err)
-	}
-
-	expiryLeaf := address.TimeoutLeaf
-
-	rootHash := expiryLeaf.TapHash()
-
-	return m.cfg.Signer.MuSig2CreateSession(
-		ctx, input.MuSig2Version100RC2, &addressParams.KeyLocator,
-		signers, lndclient.MuSig2TaprootTweakOpt(rootHash[:], false),
-	)
-}
-
 func (m *Manager) toPrevOuts(deposits []*deposit.Deposit,
```

Comment:
Shouldn't this method reuse `staticutil.ToPrevOuts`?

Also we have `loopin.StaticAddressLoopIn.toPrevOuts` method. I guess it should also reuse `staticutil.ToPrevOuts`, right?

Reply:
Yes right, fixed it.